setup.py

```diff
@@ -7,20 +7,20 @@
 
 
 setup(
-    name='lexibank_chechuroborrowing',
+    name='lexibank_daniellingua',
```

Comment:
having a different name than the ID below daniellingua vs. daniellinguafrancas is not permitted...